emmaa/util.py

```diff
@@ -447,15 +447,20 @@ def s3_put(
     unsigned_client :
         Whether to use an unsigned client.
     intelligent_tiering :
-        Whether to use intelligent tiering. Default is False.
+        Whether to use intelligent tiering. Default is False. If the object
```

Comment:
Should this be default true? I think we generally want to allow intelligent tiering.

Reply:
Yes, I think it should be. The falsy value was a relic from when I initially tested this and didn't want to set everything to intelligent tiering. Changed in https://github.com/indralab/emmaa/pull/270/commits/72d444f989e4b37eaf1cb9fe5ced1aaad41d988e.